miden-tx/Cargo.toml

```diff
@@ -18,8 +18,9 @@ path = "tests/integration/main.rs"
 
 [features]
 concurrent = ["miden-lib/concurrent", "miden-objects/concurrent", "miden-prover/concurrent", "std"]
-default = ["std"]
+default = ["std","testing"]
```

Comment:
`testing` wasn't a default feature before. Why is it now?

Reply:
It shouldn't be, I think it was a leftover from one of my latest tests. Removed!